registration/models.py

```diff
@@ -52,6 +52,15 @@ def isValid(self):
             return False
         return True
 
+    @property
+    def status(self):
+        now = timezone.now()
+        if self.startDate > now or self.endDate < now:
+            return "Expired"
```

Comment:
startdate > now means "the start date is in the future" - there should be a separate "Inactive" status for this case.